package.json

```diff
@@ -33,7 +37,19 @@
   "scripts": {
     "lint": "tslint 'src/**/*.{ts,tsx}'",
     "start": "node app.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "./node_modules/.bin/jest",
+    "dev": "./node_modules/.bin/jest --watch"
```

Comment:
Same as in `circle.yml`—is this needed? I thought Yarn add `./node_modules/.bin` to the path, just like NPM does. The lint command above relies on this and it’s been working fine for me.

Also, I feel like `dev` is not very clear on what the command does. Maybe `test-watch`?